src/types.ts

```diff
@@ -162,7 +162,7 @@ export function isStoppedResponse(data: unknown): data is StoppedResponse {
   );
 }
 
-export type Platform = 'Android' | 'iOS';
+export type Platform = 'android' | 'ios';
```

Comment:
Is this change safe? `Platform` is used in a few places: the test generation request and its also serialized as part of the test record. There'd be a mismatch here between test records stored before and after this change...

Reply:
I could update it so both upper and lower case are accepted. I tested all the functionality and nothing appeared broken. On the server side, I added a .lower() so that even platform types that remain upper case would work.

Reply:
Changing the type for data that gets serialized/deserialized seems at best unnecessary at worst a foot gun. The type is the type, if you have data that doesn't conform to it, change the data not the type, imo. Adding to the type union is non-breaking so is safer than this change.

Reply:
Ok, I added to the union. I viewed it more as the type we had was incorrect as when we get the platform from the API call it is all lower case, and I thought having to change it each time was inefficient. Especially since each word has a different uppercase structure so we can't just uppercase the first letter, which is why I didn't change the data. But I can change that if that's preferred.